src/components/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom";
+
+export const Footer = () => {
+  return (
+    <footer className="quiz-footer p-3 m-t-3 row-flex no-wrap">
+      <div className="contact-socials m-v-1 m-r-3">
+        <Link className="icon-btn footer-icon sub-heading p-1" to="mailto:nehaguptang.ng@gmail.com">
```

Comment:
I think, these links are having the same style with little differences in name. See if you can use `Array.map()` here.

It will decrease the number of lines of code in the file and repeated code also.

Reply:
sure will do